### PR TITLE
feat: Add student guide

### LINKED ISSUE #282: Feature:  Add a student Guide
### Type of feature

🍕 Feature

### Current behavior

none

### Suggested solution

add a guide like contribs but tailored to students

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by BekahHW**
## Description

Added a student guide that's a bit more tailored to their experience than the contributors guide.

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

closes #282
## Added to documentation?

- [ ] 📜 README.md
- [x] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: docs/opensauced-guides/students-guide/students-guide.md [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+
+### Using Your Repository Insights Page
+
+Once you have connected your repositories, you will be redirected to your Repository Insight Page. Here, you will see a dashboard with an overview of the repositories and the contributors who have contributed to them.
+
+There are three tabs that provide you with more information on these repositories:
+
+#### Repositories Dashboard
+
+If you're looking for an active project to contribute to, this tab can help you as it contains information such as activity levels, PR overview, PR velocity, spam PRs, contributors, and activity stats over the last 30 days. This information is also a good way to connect with others on the project or where to find support.
```
Comment (adiati98) [inline docs/opensauced-guides/students-guide/students-guide.md:228]: We can split the first sentence to understand it better.

```suggestion
If you're looking for an active project to contribute to, this tab can help you. It contains information such as activity levels, PR overview, PR velocity, spam PRs, contributors, and activity stats over the last 30 days. This information is also a good way to connect with others on the project or where to find support.
```

File: docs/opensauced-guides/students-guide/students-guide.md [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+
+![activity graph](../../../static/img/activity-repos-insights.png)
+
+Beyond understanding repositories, you may want to get insights into other contributors to help you learn more about how others are contributing to open source. One way to do that is through Contributor Insights.
+
+## Contributor Insights: Connecting with Contributors
+
+The [Contributor Insights](../../features/contributor-insights.md) feature helps you connect with other contributors or learn more about what others are up to in open source. With this feature, you can categorize, watch, and check out different groups of contributors within open source projects.
+
+### What You can do with Contributor Insights?
```
Comment (adiati98) [inline docs/opensauced-guides/students-guide/students-guide.md:256]: Not sure if we should have a question mark here.

```suggestion
### What You Can Do with Contributor Insights
```

File: docs/opensauced-guides/students-guide/students-guide.md [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+- **Contributor Insights**: The Contributor Insights feature helps you understand other contributors or track your own work in open source.
+
+## Conclusion
+
+Embarking on your open source journey with OpenSauced opens up a world of learning and development opportunities. It's a chance to apply your knowledge to real-world projects, collaborate with a global community, and build a portfolio that stands out to educators and employers alike. We're excited to support you on this journey and can't wait to see the contributions you'll make and the milestones you'll achieve. If you have any questions or need assistance, our community is here to help you navigate your path in open source.
+
+Here are some other resources you might find helpful in your journey:
+
+- [100 Days of OSS](../../community/100-days-of-oss.md)
+- [Intro to Open Source](https://intro.opensauced.pizza/#/)
```
Comment (adiati98) [inline docs/opensauced-guides/students-guide/students-guide.md:313]:
```suggestion
- [Intro to Open Source Course](https://intro.opensauced.pizza/#/)
```

File: docs/opensauced-guides/students-guide/students-guide.md [hunk 4]
```diff
@@ -0,0 +1,315 @@
...
+4. **Skill-Specific Exploration**: You can use workspaces to follow repositories that help you build specific skills or understand certain technologies, creating a tailored learning environment that aligns with your career path or focus.
+
+5. **Community Engagement Tracking**: By adding repositories you're interested in, you can monitor and engage with the communities behind them, understanding the dynamics of open source collaboration, which is invaluable for both personal and professional networking.
+
+
+## Repository Insights: Connecting Your Repositories
+
+OpenSauced [Repository Insights](../../features/repo-insights.md) allows you to track open source projects you are interested in. You can use Repository Insights Pages to track repositories' growth, analyze work, and connect with others.
+
+### Why Create a Repository Insights Page
```
Comment (adiati98) [inline docs/opensauced-guides/students-guide/students-guide.md:197]:
```suggestion
### Why Create a Repository Insights Page?
```

**Review by adiati98 — CHANGES_REQUESTED** [review]
@BekahHW I left reviews here. Most are for clarity purpose. 
Thank you!😊

File: docs/opensauced-guides/students-guide/students-guide.md [hunk 5]
```diff
@@ -0,0 +1,315 @@
...
+![your profile](../../../static/img/profile.png)
+
+On your profile, you will see the "Highlights" tab, which you can learn more about [in this section](#highlights-connecting-and-recognizing-contributions). Other tabs you may find useful on your profile are: 
+
+- **Contributions**: This tab includes a graph showing the number of PRs you have opened, [the velocity of your PRs](../../welcome/glossary.md), and the number of repositories you have contributed to in the last thirty days. Underneath the graph is a "Latest PRs" chart, which shows a list of current and recent PRs. This tab is helpful if you want to showcase a steady and consistent open source contribution experience.
+- **Recommendations**: This tab includes a list of open source projects that are chosen based on the programming languages you picked in the "Interests" section of your public profile. It is useful if you're looking for new projects to contribute to.
+
+:::tip
+
+Don't just share your profile, include in on things like your resume, LinkedIn, your GitHub Readme, and  your personal website to showcase your most up-to-date open source contributions.
```
[the remark this answers is not in the document]
Comment (BekahHW) [reply docs/opensauced-guides/students-guide/students-guide.md:54]: No. I mean don't just share it on social media. Add the link to your OpenSauced Profile on your resume, LI, GH Readme, and site. I'll work on the wording.

File: docs/opensauced-guides/students-guide/students-guide.md [hunk 6]
```diff
@@ -0,0 +1,315 @@
...
+
+#### Highlighting Dev.to Blog Posts
+
+1. **Describe the topic**: Begin with an interesting blog post summary to help others to understand your writing.
+2. **Discuss the inspiration**: Explain why you wrote this blog post. Is it to guide others on improving their coding skills, raise awareness about an issue in the tech community, or display your progress in learning a new programming language?
+3. **End with a call-to-action**: Encourage others to read and engage with the post.
+
+## Creating Your Workspace
+
+As a student working on multiple projects or contributing to various open source repositories, keeping track of progress, goals, and collaboration can be daunting. To keep track of the repositories that are most important to you or repositories that you're working on with a team, OpenSauced Workspaces can be your go-to tool for organizing and understanding your open source journey.
```
[the remark this answers is not in the document]
Comment (BekahHW) [reply docs/opensauced-guides/students-guide/students-guide.md:126]: I think the , is fine where it is bc it's modifying student here, describing the kind of student.

File: docs/opensauced-guides/students-guide/students-guide.md [hunk 7]
```diff
@@ -0,0 +1,315 @@
...
+
+To create a workspace, start by accessing the **Workspace switcher** in the sidebar. From there, you can create a new workspace or view an existing one.
+
+![workspace switcher](../../../static/img/workspace-switcher.png)
+
+<br/>
+
+:::note
+
+Whenever you select "Home", you'll be taken to the last workspace you opened.
```
[the remark this answers is not in the document]
Comment (BekahHW) [reply docs/opensauced-guides/students-guide/students-guide.md:147]: Normally that , would be inside the quotes, but for clarity here bc it's referring to a specific word they need to see, I think it's clearer to have it outside.

File: docs/opensauced-guides/students-guide/students-guide.md [hunk 8]
```diff
@@ -0,0 +1,315 @@
...
+
+:::tip
+
+To get the full benefit of your profile, share it on social media, include a link to it on your resume, LinkedIn, your GitHub Readme, and your personal website to showcase your most up-to-date open source contributions.
+
+:::
+
+### Your Dev Card
+
+Another fun way to share your open source contributions is through your [Dev Card](../../features/dev-card.md). Your Dev Card is a visual representation of your open source contributions. It includes your profile picture, name, bio, and a list of your latest contributions.
```
Comment (adiati98) [inline docs/opensauced-guides/students-guide/students-guide.md:60]: I've read through one more time, a nd this one is the last thing I've found. 

```suggestion
Another fun way to share your open source contributions is through your [Dev Card](../../features/dev-card.md). Your Dev Card is a visual representation of your open source contributions. It includes your profile picture, name, bio, and the number of your latest contributions. 
```

**Review by adiati98 — APPROVED** [review]
@BekahHW just one more thing here and this PR is good to go.
I approved the PR so you can go ahead and merge it in! Thank you! 🚀

**Pull request merged**